src/main/java/com/google/api/codegen/transformer/StaticLangApiMethodTransformer.java

```diff
@@ -808,7 +808,9 @@ private RequestObjectParamView generateRequestObjectParam(
     String transformParamFunctionName = null;
     if (context.getFeatureConfig().useResourceNameFormatOption(fieldConfig)
         && fieldConfig.requiresParamTransformation()) {
-      transformParamFunctionName = namer.getResourceOneofCreateMethod(typeTable, fieldConfig);
+      if (!fieldConfig.requiresAnyParamTransformation()) {
```

Comment:
Perhaps a comment here indicating why we don't need to apply the transformation in the Any case, because the setter will accept the supplied resource name without transformation.